cmd/goal/accountsList.go

```diff
@@ -231,6 +231,21 @@ func (accountList *AccountsList) outputAccount(addr string, acctInfo v1.Account,
 		}
 		fmt.Printf("]")
 	}
+	if len(acctInfo.AppParams) > 0 {
+		fmt.Printf("\t[created apps:")
+		for aid := range acctInfo.AppParams {
+			fmt.Printf(" %d", aid)
```

Comment:
Oh ok, I see. 
Yes, I'd sperate it with commas.

Reply:
I'm copying exactly what we do for assets. Do we really want commas for apps but spaces for assets?

Reply:
I think we want commas for both. 
Maybe a good opportunity to fix it.

Reply:
alright, did commas and ID for both